src/app/policies/components/PolicyList.vue

```diff
@@ -0,0 +1,253 @@
+<template>
+  <AppView>
+    <template #title>
+      <h2>
+        Policies: <RouteTitle
+          :title="t('policies.routes.items.title', { name: policyType.name })"
+          :render="true"
+        />
+      </h2>
+    </template>
+
+    <div class="policy-list-content">
+      <KCard data-testid="policy-type-list">
+        <template #body>
+          <div
+            v-for="(policyTypeWrapper, index) in sortedPolicyTypeWrappers"
+            :key="index"
+            class="policy-type-link-wrapper"
+            :class="{
+              'policy-type-link-wrapper--is-active': policyTypeWrapper.policyType.path === props.currentPolicyTypePath,
+            }"
+          >
+            <RouterLink
+              class="policy-type-link"
+              :to="{
+                name: 'policies-list-view',
+                params: {
+                  mesh: route.params.mesh,
+                  policyPath: policyTypeWrapper.policyType.path,
+                },
+              }"
+              :data-testid="`policy-type-link-${policyTypeWrapper.policyType.name}`"
+            >
+              {{ policyTypeWrapper.policyType.name }}
+            </RouterLink>
+
+            <div class="policy-count">
+              {{ policyTypeWrapper.numberOfPolicies }}
+            </div>
+          </div>
+        </template>
+      </KCard>
+
+      <KCard class="policy-list">
+        <template #body>
+          <div class="stack">
+            <KAlert
+              v-if="policyType.isExperimental"
+              appearance="warning"
+            >
+              <template #alertMessage>
+                <p>
+                  <strong>Warning</strong> This policy is experimental. If you encountered any problem please open an
+                  <a
+                    :href="env('KUMA_FEEDBACK_URL')"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >issue</a>
+                </p>
+              </template>
+            </KAlert>
```

Comment:
Should the instead of being an alert could be a badge next to the docs button?

Reply:
For the length of the content, I really prefer this being a KAlert. We can move it into the toolbar so it’s next to the docs button though.

Reply:
This badge content could be just `beta`

Reply:
Done and moved.